src/main/java/htsjdk/tribble/readers/TabixReader.java

```diff
@@ -34,19 +34,22 @@
 import java.io.InputStream;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.nio.channels.SeekableByteChannel;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Function;
 
 /**
  * @author Heng Li <hengli@broadinstitute.org>
  */
 public class TabixReader {
-    private String mFn;
-    private String mIdxFn;
-    private BlockCompressedInputStream mFp;
+    private final String mFn;
+    private final String mIdxFn;
+    private final Function<SeekableByteChannel, SeekableByteChannel> mIdxWrpr;
```

Comment:
Erm, no need to obey the questionable preference of this class for abbreviated names -- name it something humans can pronounce like `mIndexWrapper`